AmazonChimeSDK/AmazonChimeSDK/session/DefaultMeetingSession.swift

```diff
@@ -49,6 +49,23 @@ import Foundation
         let activeSpeakerDetector =
             DefaultActiveSpeakerDetector(audioClientObserver: audioClientObserver,
                                          selfAttendeeId: configuration.credentials.attendeeId)
+
+        let contentModality = String(DefaultModality.separator) + ModalityType.content.description
+        let contentShareCredentials = MeetingSessionCredentials(
+            attendeeId: configuration.credentials.attendeeId + contentModality,
```

Comment:
nit: should we use string interpolation?

Reply:
since this is directly concatenating two strings, this is preferred:
https://docs.swift.org/swift-book/LanguageGuide/StringsAndCharacters.html